### ISSUE TITLE
How to use Keyboard and KeyboardButton?

### ISSUE THREAD

**Issue by koteezy** [botman/botman#841]
How to use Keyboard and KeyboardButton with ask method in Conversation?

Can you please provide a example?

(telegram driver)

**Comment by koteezy** [comment]
We can use:
`
Keyboard::create()->type( Keyboard::TYPE_KEYBOARD )
                       ->oneTimeKeyboard()
                       ->addRow( KeyboardButton::create( 'Hello world' ) )
                       ->toArray();
`
in reply() method, but, can i use it in ask() method?

**Comment by christophrumpel** [comment]
Yeah you can use it in ask as well, both methods work with messages and templates. If it doesn't work, it must be something else. Do you get any errors?

**Comment by koteezy** [comment]
@christophrumpel I remember that there were no errors, but there was nothing in the telegram, too, I will try to return to the project the other day, and see if I fail - just close the issue. Thanks for the reply.

**Comment by wikreate0398** [comment]
I tried to make this request, but keyboard didn't worked

return $this->ask($question, function (Answer $answer) { 
             // some logic
        }, Keyboard::create()->type('keyboard') ->oneTimeKeyboard() ->addRow( 
                 KeyboardButton::create( 'Hello world' ) ) ->toArray());
    }
